daemon/daemon.go

```diff
@@ -105,7 +105,8 @@ func (c *Command) canAccess(r *http.Request, user *auth.UserState) bool {
 		if c.PolkitOK != "" {
 			allow, err := strconv.ParseBool(r.Header.Get(client.AllowInteractionHeader))
 			if err != nil {
-				logger.Noticef("error parsing %s header: %s", client.AllowInteractionHeader, err)
+				// default behaviour if the header
+				// cannot be parsed
```

Comment:
@mvo5 That is ignoring _all_ errors, including improper headers. The logging as done in the original PR seems fine for cases which are not related to a missing header. I believe the Get will return "" in those.